### DISCUSSION TITLE
Sive.Host would like to host demo.boxbilling.org

### DISCUSSION THREAD

**Discussion by sivehost** [boxbilling/boxbilling#1593] [Q&A]
Hello,
We are a Web Hosting and Cloud Computing company based in South Africa called Sive.Host. We have been in business for close to 14 years this year, and we would like to host the demo platform for demo.boxbilling.org for free. Not sure who controls the demo.boxbilling.org A-record, perhaps they can reach out to us to get it going?

**Comment by BRGustavoRibeiro** [comment]
BoxBilling is, unfortunately, dead unmaintained software - I truly believe it should be archived now, but for some reason it still is available.

[@FOSSBilling ](https://github.com/FOSSBilling/FOSSBilling) is the _de facto_ BoxBilling continuation. Feel free to check that out.

**Reply by sivehost** [reply]
Looks promising too. Could we host the demo of it for you as our contribution?

**Reply by jaapmarcus** [reply]
Demo is currently sponsored by my self.. And I really don't mind covering the costs for it so it is really not needed..

**Reply by sivehost** [reply]
That is great. Let us know in the future if ever there is a need.
Thank you for your prompt responses.

**Reply by ghost** [reply]
> Demo is currently sponsored by my self.. And I really don't mind covering the costs for it so it is really not needed..

is there any effort to revive BB? 
I now have a team working with me and BB is an excellent option for our team

**Reply by jaapmarcus** [reply]
Boxbilling is dead. Fossbilling is a bit more active but also there a lot of issues with availble time..
